### ISSUE TITLE
The Fizz is supposed to give us an item to charge, but after acc...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#7599]
> The Fizz is supposed to give us an item to charge, but after accepting the quest we did not get any quest items. THe quest is not possible to complete without it.

.go xyz 6364.310000 -4818.320000 732.664000 1 5.732430

**Comment by Shadownflame** [comment]
Some duplicate reports suggest the quest does award credit even without the item if the player gets hit by the target ability.

**Comment by ghost** [comment]
Working as intended, you're not supposed to get an item

**Comment by Shadownflame** [comment]
> Working as intended, you're not supposed to get an item

"Here's the tool fer the job.

<Tanum hands you a dangerous looking device covered in buttons, switches and lights.>"
From the quest text. If it was redesigned to work without an item the description and the instruction both needs changing.

I do get quest credit for being hit by the mobs lightning even without the device but this is terribly confusing to players.

There is also a noteworthy complaint of players being able to complete this quest off one single Chimera repeatedly casting the breath while they heal and afk. It would suit typical classic quest design better if they could only charge off one cast per chimera.

**Comment by Bennylavaa** [comment]
I agree the quest basically makes it sound like you are getting some item, so either the quest needs rewriting or a item needs to be given to charge up for the quest. 

As for the afk thing a item with some sort of on use could solve this, that or make the npc debuff themselves so it will only trigger for 1 per.
